dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageErrorCause.java

```diff
@@ -20,12 +20,16 @@
 
 /** Key to group linkage errors by their causes. */
 @AutoValue
+// TODO "cause" is unclear. Is it the target or the source? I don't know.
+// Rename things to remove the word "cause" and use more specific terminology.
 abstract class LinkageErrorCause {
 
   /** Returns the reason for the error */
   abstract SymbolNotResolvable.Reason getReason();
 
-  /** Returns the symbol causing the error. It's either class name, field name, or method name. */
+  /** 
+   * Returns the fully package qualified name of the missing or inaccessible symbol.
+   * This is a class name, field name, or method name. */
```

Comment:
For methods, does this include the parameter types?

Reply:
no, it does not